src/utils/metakg/config.py

```diff
@@ -0,0 +1 @@
+SMARTAPI_URL = "https://smart-api.info/api/query?q=tags.name:translator&size=500&fields=paths,servers,tags,components.x-bte*,info,_meta&meta=1"
```

Comment:
should just use SmartAPI model to get the API metadata directly, not from this URL, since we are now in the SmartAPI application itself.

Reply:
oh ok, I just don't know they are the same. I will update my PR.